cmdstanpy/utils.py

```diff
@@ -887,21 +897,45 @@ def install_cmdstan(
     if dir is not None:
         cmd.extend(['--dir', dir])
     if overwrite:
-        cmd.extend(['--overwrite', 'TRUE'])
+        cmd.extend(['--overwrite'])
     if verbose:
-        cmd.extend(['--verbose', 'TRUE'])
+        cmd.extend(['--verbose'])
     proc = subprocess.Popen(
         cmd, stdout=subprocess.PIPE, stderr=subprocess.PIPE, env=os.environ
     )
-    while proc.poll() is None:
-        print(proc.stdout.readline().decode('utf-8').strip())
+    if platform.system() != 'Windows':
```

Comment:
Why this is blocked on Windows?

Reply:
will remove that - was trying to see if that would fix Windows build problem - SO suggested that reading from stdout would cause pipes to block and processes to hang.  since unit tests continued to fail on Windows, I don't think this was the culprit.  however, if we can get rid of chain of subprocesses, that would be best.